ext/net/lib.rs

```diff
@@ -19,10 +23,29 @@ use std::sync::Arc;
 
 pub const UNSTABLE_FEATURE_NAME: &str = "net";
 
+#[derive(Clone, Eq, PartialEq, Hash, Debug)]
+pub struct NetPermissionHost {
+  pub host: Host,
+  pub port: Option<u16>,
+}
+
+impl NetPermissionHost {
+  pub fn from_str(host: &str, mut port: Option<u16>) -> Result<Self, AnyError> {
```

Comment:
I think clippy will complain about the name of this method. I suggest to rename it to `from_host_and_maybe_port`. Also `port` should be a mutable variable, do a copy if you need to change it

Reply:
ok